Applications/spire/include/spire/login_ui_tester/login_ui_tester.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef SPIRE_LOGIN_UI_TESTER_HPP
+#define SPIRE_LOGIN_UI_TESTER_HPP
+#include <QEvent>
+#include <QPushButton>
+#include <QWidget>
+#include "spire/login/login_window.hpp"
```

Comment:
Just include "spire/login/login.hpp" instead of login_window.hpp
Also only include "spire/ui/ui.hpp" instead of window.hpp